Doppler.BillingUser/Infrastructure/BillingRepository.cs

```diff
@@ -808,6 +808,37 @@ [dbo].[BillingCredits] BC
             return billingCredit;
         }
 
+        public async Task<AccountingEntry> GetInvoice(int idClient, string paymentId)
+        {
+            var connection = _connectionFactory.GetConnection();
+            var invoice = await connection.QueryFirstOrDefaultAsync<AccountingEntry>(@"
+SELECT
+    AE.[Date],
+    AE.[Amount],
+    AE.[Status],
+    AE.[Source],
+    AE.[AuthorizationNumber],
+    AE.[InvoiceNumber],
+    AE.[AccountEntryType],
+    AE.[AccountingTypeDescription],
+    AE.[IdClient],
+    AE.[IdAccountType],
+    AE.[IdInvoiceBillingType],
+    AE.[IdCurrencyType],
+    AE.[CurrencyRate],
+    AE.[Taxes]
+FROM
+    [dbo].[AccountingEntry] AE
+WHERE
+    idClient = @idClient AND AuthorizationNumber = @paymentId",
+        new
```

Comment:
This doesn't seem to be well indented.

Reply:
@afantini you are talking about the last line, right?

```suggestion
                new
```

Reply:
Yes! sorry for the delay.

Reply:
No problem! you were right!. I modified it that before merging